operator/deploy/kiali/kiali_cr.yaml

```diff
@@ -399,10 +396,7 @@ spec:
 #        username: ""
 #      enabled: true
 #      in_cluster_url: ""
-#      namespace: ""
 #      namespace_selector: true
-#      port: 16686
-#      service : ""
```

Comment:
remove the corresponding default settings from the default/main.yaml here: https://github.com/kiali/kiali/blob/v1.10/operator/roles/kiali-deploy/defaults/main.yml#L121-L124

Reply:
this has been done.